src/isaacs_mapping/launch/isaacs_mapping.launch

```diff
@@ -1,19 +1,22 @@
 <launch>
-  
-  <!-- ROSBRIDGE -->
-  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>
+  <!-- <remap from="/voxblox_node/mesh" to="/voxblox_node/mesh2"/> -->
+  <!-- ROSBRIDGE 
+  	<include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch"/>
+  -->
```

Comment:
If we keep this, we should mention it in our documentation and keep things consistent since we're currently telling people to run stuff in terminal

Reply:
Maybe we can also include 2 isaacs_mapping launch files, one for running with the ZED camera and one for running with ROS.